eox_tenant/signals.py

```diff
@@ -225,8 +229,10 @@ def start_async_tenant(sender, *args, **kwargs):  # pylint: disable=unused-argum
     if not getattr(base_settings, "USE_EOX_TENANT", False):
         return
 
-    headers = sender.request.get('headers') or {}
+    context = sender.request
+    headers = context.get('headers') or {}
```

Comment:
can we change this to `headers = context.get('headers', {})` ?
everything else lgtm

Reply:
I think what this does is that it also removes the case where `sender.request.get('headers') = None`.
We would have:
` None or {}` which resolves to {}